jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-mulitenant-eks.jenkinsfile

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'k8s-eks',
+    region: 'eu-north-1',
+    test_name: 'longevity_operator_multi_tenant_test.LongevityOperatorMulitiTenantTest.test_custom_time',
+    test_config: 'test-cases/scylla-operator/longevity-scylla-operator-3h-multitenant.yaml',
+    email_recipients: 'julia.yakovlev@scylladb.com',
```

Comment:
remove the email

Reply:
changed to 'qa@scylladb.com,scylla-operator@scylladb.com'